pyproject.toml

```diff
@@ -3,3 +3,117 @@ requires = ["setuptools>=64", "setuptools_scm>=8"]
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools_scm]
+
+[project]
+name = "stonesoup"
+description = "A tracking and state estimation framework"
+maintainers = [
+    { name = "Defence Science and Technology Laboratory UK", email = "stonesoup@dstl.gov.uk" },
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3 :: Only",
+    "Topic :: Scientific/Engineering",
+]
+requires-python = ">=3.9"
+dependencies = [
+    "matplotlib",
+    "mergedeep",
+    "numpy>=1.17",
+    "ordered-set",
+    "pymap3d",
+    "ruamel.yaml>=0.16.5",
+    "rtree",
+    "scipy",
+    "utm",
+]
+dynamic = ["version"]
+
+[project.readme]
+file = "README.md"
+content-type = "text/markdown"
+
+[project.license]
+text = "MIT"
+
+[project.urls]
+Homepage = "https://github.com/dstl/Stone-Soup"
+
+[project.optional-dependencies]
+dev = [
+    "confluent-kafka",
+    "coverage>=7.2",
+    "flake8",
+    "folium",
+    "h5py",
+    "pandas",
+    "ortools",
+    "pillow",
+    "plotly>=5.0",
```

Comment:
Given we have most documentation using (Animated)Plotterly should plotly dependency move from dev optional dependencies to the required dependencies?